modules/aioseop_opengraph.php

```diff
@@ -1884,6 +1884,13 @@ function settings_update() {
 		 * @param string $hook_suffix
 		 */
 		public function admin_enqueue_scripts( $hook_suffix ) {
+			global $current_screen;
+			
```

Comment:
1888 | ERROR | [x] Whitespace found at end of line
      |       |     (Squiz.WhiteSpace.SuperfluousWhitespace.EndLine)